CMakeLists.txt

```diff
@@ -792,3 +791,27 @@ foreach(file ${ZIG_STD_FILES})
     get_filename_component(file_dir "${ZIG_STD_DEST}/${file}" DIRECTORY)
     install(FILES "${CMAKE_SOURCE_DIR}/std/${file}" DESTINATION "${file_dir}")
 endforeach()
+
+include(CheckCXXSourceCompiles)
+include(CTest)
+
+set(fuzzer_flags "-fsanitize=address,undefined,fuzzer")
+set(CMAKE_REQUIRED_FLAGS "${fuzzer_flags}")
+check_cxx_source_compiles("
+#include <stdint.h>
+#include <stdlib.h>
```

Comment:
This is for size_t, right?  Technically, that's from stddef.h.

Reply:
OK ya the two headers are for `size_t` and `uint8_t`. Will try to replace `stdlib` with `stddef`.